pkg/model/storage_spec.go

```diff
@@ -20,6 +20,9 @@ type StorageSpec struct {
 	// Source URL of the data
 	URL string `json:"URL,omitempty"`
 
+	// URL of the git Repo to clone
+	Repo string `json:"Repo,omitempty"`
+
```

Comment:
> We should also start looking into defining the inputs as URIs (ipfs://, http://, git://, s3://...) instead of different input arguments, but that can be a separate change
I think just having one flag is better as Walid stated but that's a separate thing